diff -durN libxode.orig/src/pool.c libxode/src/pool.c
--- libxode.orig/src/pool.c	Mon Oct 30 09:52:11 2000
+++ libxode/src/pool.c	Wed Dec 19 21:09:20 2001
@@ -1,5 +1,3 @@
-<<<<<<< pool.c
-=======
 /*
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
@@ -98,21 +96,7 @@
 /* mem should always be freed last */
 void _pool_cleanup_append(pool p, struct pfree *pf)
 {
-<<<<<<< pool.c
-<<<<<<< pool.c
-    struct pfree *ret, *cur;
-
-    /* make the storage for the tracker */
-    ret = malloc(sizeof(struct pfree));
-    ret->f = f;
-    ret->arg = arg;
-    ret->next = NULL;
-=======
     struct pfree *cur;
->>>>>>> 1.10
-=======
-    struct pfree *cur;
->>>>>>> 1.10
 
     if(p->cleanup == NULL)
     {
@@ -307,12 +291,7 @@
 {
     return;
 }
-<<<<<<< pool.c
 
 /********************************************/
 #endif /* POOLDEBUG */
 /********************************************/
->>>>>>> 1.8
-=======
-#endif
->>>>>>> 1.15
